node/src/accountant/payable_dao.rs

```diff
@@ -981,54 +989,48 @@ mod tests {
         //Two accounts differ only in debt's age but not balance which allows to check doubled ordering,
         //by balance and then by age.
         let now = now_time_t();
-        let timestamp1 = now - 70_000;
-        let timestamp2 = now - 55_120;
-        let timestamp3 = now - 200_001;
-        let timestamp4 = now - 19_999;
-        let timestamp5 = now - 30_786;
-        let timestamp6 = now - 100_401;
-        let timestamp7 = now - 80_333;
-        let main_setup = |insert: &dyn Fn(&str, i128, i64, Option<i64>)| {
+        let main_setup = |conn: &dyn ConnectionWrapper, insert: InsertPayableHelperFn| {
```

Comment:
`insert` is already a closure, and might be able to close around the connection so that another layer of closure wasn't needed.